pkg/services/store/entity/README.md

```diff
@@ -137,4 +137,14 @@ GF_GRAFANA_APISERVER_STORAGE_TYPE=unified-grpc ./bin/grafana server
 You can then list the previously-created playlists with:
 ```sh
 kubectl --kubeconfig=./grafana.kubeconfig get playlist
-```
\ No newline at end of file
+```
+
+## Changing protobuf interface
+
+- install [protoc](https://grpc.io/docs/protoc-installation/)
+- install the protocol compiler plugin for Go
+```sh
+go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+```
+- make changes in `.proto` file
+- run `./pkg/services/store/entity/generate.sh` to regenerate definitions in this package.
```

Comment:
`make protobuf` will handle all proto definitions (by running the various generate.sh scripts), should we just have folks use that vs calling individual scripts directly?

Reply:
True. Thanks for the comment. The reason why I mentioned individual script here instead of `make protobuf` is to run the codegen [in this only package rather than across entire codebase](https://github.com/grafana/grafana/blob/3e4abdeb18de74d1a9f5472e199ead53faf52e0b/Makefile#L305)
I don't have strong opinion reg this.

Reply:
if the rest of the protos are up to date it should be a no-op (modulo protoc version differences) so my thinking was to make sure things are consistent and up to date vs trying to optimize for targeting just this package. It's less relevant now that we aren't pulling the grn proto into this one, but with how fast they generate it seems like a small amount of overhead to have a more consistent developer experience.

Reply:
Thanks for explaining, I see now. Very good point.